src/main/java/com/igot/cb/CbCommunityServiceApplication.java

```diff
@@ -0,0 +1,47 @@
+package com.igot.cb;
+
+import org.apache.http.client.config.RequestConfig;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.web.client.RestTemplate;
+
+
+/**
+ * @author Mahesh RV
+ * @author Ruksana
+ */
+@EnableJpaRepositories(basePackages = {"com.igot.cb.*"})
+@ComponentScan(basePackages = "com.igot.cb")
+@EntityScan("com.igot.cb")
+@SpringBootApplication
+public class CbCommunityServiceApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(CbCommunityServiceApplication.class, args);
+	}
+
+    @Bean
+    public RestTemplate restTemplate() {
+        return new RestTemplate(getClientHttpRequestFactory());
+    }
+
+    private ClientHttpRequestFactory getClientHttpRequestFactory() {
+        int timeout = 45000;
```

Comment:
is it possible to have this as configuration ??

Reply:
done